examples/src/main/java/com/traveltime/examples/TimeFilterExample.java

```diff
@@ -0,0 +1,77 @@
+package com.traveltime.examples;
+
+import com.traveltime.sdk.TravelTimeSDK;
+import com.traveltime.sdk.auth.TravelTimeCredentials;
+import com.traveltime.sdk.dto.common.Coordinates;
+import com.traveltime.sdk.dto.common.Location;
+import com.traveltime.sdk.dto.common.Property;
+import com.traveltime.sdk.dto.requests.TimeFilterFastRequest;
+import com.traveltime.sdk.dto.requests.timefilterfast.ArrivalSearches;
+import com.traveltime.sdk.dto.requests.timefilterfast.transportation.Driving;
+import com.traveltime.sdk.dto.requests.timefilterfast.OneToMany;
+import lombok.val;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static io.vavr.API.*;
+import static io.vavr.Patterns.$Left;
+import static io.vavr.Patterns.$Right;
+
+/**
+ * Example showing how to find 3 closest shops by travel time given a list of shops with locations.
+ * List is generated randomly. Here we are using driving transportation mode, but you can use different ways of
```

Comment:
s/List is generated randomly./The list of shop coordinates is generated randomly./ 

The choice of departure coordinates might be also explained.